pkg/tuned/tuned.go

```diff
@@ -313,9 +314,18 @@ func disableSystemTuned() {
 	}
 }
 
+func profilesEqual(profileFile string, profileData string) bool {
+	content, err := ioutil.ReadFile(profileFile)
+	if err != nil {
+		content = []byte{}
+	}
+
+	return profileData == string(content)
+}
+
 // profilesExtract extracts TuneD daemon profiles to the daemon configuration directory.
-// If the data in the to be extracted recommended profile is different than the current
-// recommended profile, the function returns true.
+// If the data in the to be extracted recommended profile or the profiles being included
```

Comment:
I'm not sure how to read this comment, is it like this?
> If the data in the to-be-extracted-recommended-profile

if yes, then the `-` helps a lot, if no ... the comment should be reformulated :)

Reply:
Your interpretation is correct. I'll add the `-`, thank you!